hoomd/md/nlist.py

```diff
@@ -185,51 +195,6 @@ def detach(self):
 
 
 class stencil(nlist):
```

Comment:
Sorry I didn't want the docstrings deleted but the documentation removed from the sphinx documentation builds.